README.md

```diff
@@ -1,2 +1,23 @@
 # HW01
-Edit the readme
+Nabilah Sammudin
+7/6/2020
+
+## Brief Biography
+
+My name is **Nabilah** and I am from **Malaysia**. I graduated from **Purdue University** back in 2017 and I am currently working as a research technician in the same lab I did my research as an undergraduate. 
+
+I am interested in studying *Learning and Memory* at the *cell, molecular level* as well as potentially using *computational techniques* in my research. Although I have very little background in coding, I believe that it is an invaluable skill in doing my PhD and I am excited to learn more through this class!
+
+### Hobbies
+
+* Baking 
+* Cooking
+* Reading
+* Hiking
+
+I recently stumbled across this beautiful sunset picture and it is my current favorite image.
+
+![Sunset on the Rocks in Malaysia](https://asiasociety.org/files/uploads/303images/121108_POD_malaysia.jpg)
```

Comment:
It looks like you have the same issue here that I had.  I'm not sure why the image caption is not showing up in the html version but you have it written in as I thought properly.  I'm going to try to look into this more and see if there is another format or something we are missing.